src/test/java/com/google/javascript/clutz/partial/test.js

```diff
@@ -0,0 +1,32 @@
+goog.module("missing.imported.base");
```

Comment:
This file needs a better name than 'test.js'. I think it is testing a goog.require aliasing inside a goog.module.

Reply:
I've renamed it to `es6_missing_imported_base.js` to reflect that it's the same as `missing_imported_base.js`, but using es6 classes instead of constructor annoations.